koku/api/settings/tags/mapping/utils.py

```diff
@@ -58,4 +64,18 @@ def resummarize_current_month_by_tag_keys(enabled_rows, schema_name):
             )
             delayed_summarize_current_month(schema_name, list(provider_uuids), provider_type)
 
-    return provider_type
+
+def retrieve_tag_rate_mapping(schema_name):
+    """Retrieves the tag rate mapping."""
+    tag_rate_map = get_cached_tag_rate_map(schema_name)
+    if tag_rate_map:
+        return tag_rate_map
+    ocp_cost_models = CostModel.objects.filter(source_type=Provider.PROVIDER_OCP)
+    tag_rate_mapping = {}
+    for cost_model in ocp_cost_models:
+        for rate in cost_model.rates:
+            if tag_rate := rate.get("tag_rates"):
+                tag_key = tag_rate.get("tag_key")
```

Comment:
Will there _always_ be a `tag_key` in `tag_rate`?

Reply:
Yes, the `TagRateSerializer` requires the tag_key when creating a cost model.